### PR TITLE
restructure project to make it deployable again

### PR DESCRIPTION
**Pull request by JonasJurczok**
@MALPI  here is the proposal I talked to you about.
I dropped the modules and integrated the sample in the normal tests.

This will make it deployable and usable again

### CODE REVIEW

**Comment by MALPI** [comment]
will take a look during the day

File: src/test/java/org/zalando/failsafeactuator/sample/SampleEndpointTest.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  *
  * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (MALPI) [inline src/test/java/org/zalando/failsafeactuator/sample/SampleEndpointTest.java]: Can you drop license header here?

File: sample/src/main/java/org/zalando/failsafeactuator/sample/SampleApplicationBanner.java [hunk 2]
```diff
@@ -1,27 +0,0 @@
-package org.zalando.failsafeactuator.sample;
```
Comment (MALPI) [inline sample/src/main/java/org/zalando/failsafeactuator/sample/SampleApplicationBanner.java]: Can you recover the Banner? I actually liked that.

File: pom.xml [hunk 3]
```diff
@@ -5,23 +5,21 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.zalando</groupId>
-    <artifactId>failsafe-actuator-parent</artifactId>
-    <version>0.5.0</version>
+    <artifactId>failsafe-actuator</artifactId>
+    <version>0.5.5</version>
```
Comment (MALPI) [inline pom.xml]: Should be 0.5.3. right?

**Comment by JonasJurczok** [comment]
Yeah.. the problem with the sample being its own module is that you either have a separate repository in github (the clean way) or you have a maven submodule that is not connected to the module tree (the hacky way). As it is just three files I would highly recommend to leave it where it is now.. It is still readable/findable and usable and you save yourself a lot of maintenance overhead for the extra project.

File: sample/src/main/java/org/zalando/failsafeactuator/sample/SampleApplicationBanner.java (hunk 2 above)
Comment (coders-kitchen) [reply sample/src/main/java/org/zalando/failsafeactuator/sample/SampleApplicationBanner.java]: Yes please :)

**Comment by coders-kitchen** [comment]
👍

**Comment by MALPI** [comment]
:+1:

**Pull request merged**